src/components/form/validation.js

```diff
@@ -19,20 +19,21 @@ const Validation = {
     return value && counter < 1 ? undefined : msg;
   },
   validGithub: (value) => {
-    const re = /https:\/\/github\.com\/[A-z0-9_-]+\/?/;
+    const re = /https:\/\/(www\.)?github\.com\/[A-z0-9_-]+\/?/;
```

Comment:
both github and twitter have redirects so their urls never have www, which is the reason why I didn't put them in the first place.

Reply:
Well, let their servers do the job then. www. Is still valid